com.unity.mobile.notifications/Runtime/Android/AndroidNotification.cs

```diff
@@ -208,6 +213,7 @@ public AndroidNotification(string title, string text, DateTime fireTime)
             GroupAlertBehaviour = GroupAlertBehaviours.GroupAlertAll;
             ShowTimestamp = false;
             ShowCustomTimestamp = false;
+            ShowInForeground = true;
```

Comment:
This assignment only applies to this constructor, but struct also has a default no arg one you cannot write. So for default-constructed struct the value would be false.
I suggest instead to have a private variable m_SilentInForeground in struct with ShowInForeground returning/setting the opposite value from that. Then the property would have the correct value and backward compatibility is preserved.

Reply:
Thanks for taking a look @aurimasc. I introduced m_SilentInForeground to back ShowInForeground public interface as you suggested. Please let me know if further changes are necessary